File: curso_c_cpp/c/06_bucles/ejemplos/tabla_anidada.c
/*
 * Bucles anidados: tabla de multiplicar del 1 al 5.
 */
#include <stdio.h>

int main(void) {
    for (int i = 1; i <= 5; i++) {
        for (int j = 1; j <= 5; j++) {
            printf("%3d", i * j);    // %3d alinea en ancho 3
        }
        printf("\n");                // nueva fila
    }
    return 0;
}

File: curso_java/06_bucles/ejemplos/TablaAnidada.java
/*
 * Bucles anidados: tabla de multiplicar del 1 al 5.
 */
public class TablaAnidada {
    public static void main(String[] args) {
        for (int i = 1; i <= 5; i++) {
            for (int j = 1; j <= 5; j++) {
                System.out.printf("%4d", i * j);   // %4d alinea en ancho 4
            }
            System.out.println();                   // nueva fila
        }
    }
}
